tests/cypress/latest/support/commands.ts

```diff
@@ -433,9 +456,18 @@ Cypress.Commands.add('checkChart', (operation, chartName, namespace, version, qu
     checkApiStatus();
   }
 
-  // Resource should be deployed (green badge)
-  cy.get('.outlet').contains('Deployed', { timeout: 180000 });
-  cy.namespaceReset();
+  if (operation == 'Install') {
+    // All resources (usually an App and its CRD) in the namespaces are deployed (green badge)
+    cy.contains('Installed Apps').should('be.visible');
+    cy.setNamespace(namespace);
+    cy.waitForAllRowsInState('Deployed', 180000);
+    cy.namespaceReset();
+  }
+
+  if (operation == 'Update') {
+    cy.contains(new RegExp('Installed App:.*Deployed'), { timeout: 120000 }).should('be.visible');
+    cy.waitForAllRowsInState('Active', 120000);
```

Comment:
```suggestion
    cy.waitForAllRowsInState('Active');
```
Default timeout already set